nuxeo-wopi/src/main/java/org/nuxeo/wopi/WOPIServiceImpl.java

```diff
@@ -70,19 +75,45 @@
 
     protected PublicKey oldProofKey;
 
+    protected String discoveryURL;
+
     @Override
     public void start(ComponentContext context) {
-        Path discoveryPath = Paths.get(Environment.getDefault().getData().getAbsolutePath(), WOPI_DIR, DISCOVERY_XML);
-        if (Files.notExists(discoveryPath)) {
-            log.error("Discovery file does not exist, WOPI disabled.");
-            log.debug("Discovery file path: {}", discoveryPath);
+        String wopiDiscoveryURL = Framework.getProperty(WOPI_DISCOVERY_URL_PROPERTY);
+        if (wopiDiscoveryURL == null) {
+            log.warn("No WOPI discovery URL configured, WOPI disabled.");
```

Comment:
Maybe add something like "you can configure the nuxeo.wopi.discoveryURL property to enable WOPI"